File: Modul5/C/PRAK505-2310817210020-MuhammadHasbiyanRusyadi.c
#include <stdio.h>
void Biodata(int thnlahir, char Nama[], char Asal[]){
    int thnskrg = 2023;
    int umur = thnskrg - thnlahir;
    printf("\nPerkenalkan Nama Saya : %s\n", Nama);
    printf("Umur Saya : %d\n", umur);
    printf("Saya Adalah Angkatan : %d\n", thnskrg);
    printf("Asal Saya dari : %s\n\n", Asal);}
int main() {
    int thnlahir;
    char Nama[20], Asal[15];
        printf("\nTahun lahir : ");
        scanf(" %d",&thnlahir);
        printf("Namaku : ");
        scanf(" %[^\n]%*c",&Nama);
        printf("Asal : ");
        scanf(" %[^\n]%*c",&Asal);
        Biodata(thnlahir, Nama, Asal);
return 0;}

File: Modul5/Python/PRAK505-2310817210020-MuhammadHasbiyanRusyadi.py
def Biodata(thnlhr,Nama,Asal) :
    thnskrg = 2023
    Umur = thnskrg - thnlhr
    print(f"\nPerkenalkan Nama Saya : {Nama}")
    print(f"Umur saya : {Umur}")
    print(f"Saya Adalah Angkatan : {thnskrg}")
    print(f"Asal Saya dari : {Asal}\n")
thnlhr = int(input("\nTahun lahir : "))
Nama = input("Namaku : ")
Asal = input("Asal : ")
Biodata(thnlhr, Nama, Asal)
